src/core_plugins/kbn_vislib_vis_types/public/controls/gauge_options.html

```diff
@@ -0,0 +1,294 @@
+          <p>
+            <i class="fa fa-danger text-danger"></i>
+            <strong>Required:</strong> You must specify at least one range.
+          </p>
+        </div>
+
+        <div
+          ng-click="addRange()"
+          class="sidebar-item-button primary">
+          Add Range
+        </div>
+        <div class="text text-center text-info">Note: colors can be changed in the legend</div>
+      </div>
+    </div>
+  </div>
+  <div ng-show="vis.params.gauge.colorsRange.length > 1">
+    <div class="kuiSideBarCollapsibleTitle">
+      <div
+        class="kuiSideBarCollapsibleTitle__label"
+        ng-click="showColorOptions = !showColorOptions"
+      >
+      <span
+        aria-hidden="true"
+        ng-class="{
+        'fa-caret-down': showColorOptions,
+        'fa-caret-right': !showColorOptions
+        }"
+        class="fa fa-caret-right kuiSideBarCollapsibleTitle__caret"
+      ></span>
+        <span class="kuiSideBarCollapsibleTitle__text">
+        Color Options
+      </span>
+      </div>
+    </div>
+    <div ng-if="showColorOptions" class="kuiSideBarCollapsibleSection">
+      <div class="kuiSideBarSection">
+        <div class="kuiSideBarFormRow">
+          <label class="kuiSideBarFormRow__label" for="colorSchema">
+            Color Schema
+          </label>
+          <div class="kuiSideBarFormRow__control">
+            <select
+              id="colorSchema"
+              class="kuiSelect kuiSideBarSelect"
+              ng-model="vis.params.gauge.colorSchema"
+              ng-options="mode for mode in vis.type.params.colorSchemas"
+            ></select>
+          </div>
+          <div class="text-info text-center" ng-show="customColors" ng-click="resetColors()">reset colors</div>
+        </div>
+
+        <div class="kuiSideBarFormRow">
+          <label class="kuiSideBarFormRow__label" for="invertColors">
+            Reverse Color Schema
+          </label>
+          <div class="kuiSideBarFormRow__control">
+            <input class="kuiCheckBox" id="invertColors" type="checkbox" ng-model="vis.params.gauge.invertColors">
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+
+
+
+
+  <div>
+    <div class="kuiSideBarCollapsibleTitle">
+      <div
+        class="kuiSideBarCollapsibleTitle__label"
+        ng-click="showStyle = !showStyle"
+      >
+        <span
+          aria-hidden="true"
+          ng-class="{
+          'fa-caret-down': showStyle,
+          'fa-caret-right': !showStyle
+          }"
+          class="fa fa-caret-right kuiSideBarCollapsibleTitle__caret"
+        ></span>
+        <span class="kuiSideBarCollapsibleTitle__text">
+          Style
+        </span>
+      </div>
+    </div>
+    <div ng-if="showStyle" class="kuiSideBarCollapsibleSection">
+      <div class="kuiSideBarFormRow" ng-show="vis.params.gauge.colorsRange.length > 1">
+        <label class="kuiSideBarFormRow__label" for="labelColor">
+          Color Labels
```

Comment:
this is now a dropdown with 3 values: color labels, color background or color none
hope this makes it clearer ...